lib/internal/make-url-request.js

```diff
@@ -38,13 +38,19 @@ var version = require('../version');
  * @param {function(?)} onError
  * @return {function()}
  */
-module.exports = function makeUrlRequest(url, onSuccess, onError) {
+module.exports = function makeUrlRequest(url, onSuccess, onError, postData) {
```

Comment:
Yeah I added it to the end to avoid breaking that, but it definitely reads more clearly as second param. I'll change that and we can do a major version change and note the backward incompatibility.